tests/suite/test_app_protect_waf_policies.py

```diff
@@ -384,6 +385,82 @@ def test_ap_waf_policy_logs(
         assert f'request_status="blocked"' in log_contents
         assert f'outcome="REJECTED"' in log_contents
 
+    @pytest.mark.flaky(max_runs=3)
```

Comment:
Is there something that we can do to make it more stable? I don't think it's ideal to introduce a new flaky test.

Reply:
I think that mark was in there by mistake, i didn't see this test being flaky when running it. removed